source/_integrations/duckdns.markdown

```diff
@@ -29,7 +29,7 @@ duckdns:
 
 {% configuration duckdns %}
   domain:
-    description: Your duckdns subdomain (without the `.duckdns.org` suffix).
```

Comment:
I don't think this change is correct. The DuckDNS API expects a sudomain, not a full domain.

Unfortunately, this PR does not explain the reasoning behind this change in its description. Could you elaborate?